BoxContentSDK/BoxContentSDK/BOXContentSDKConstants.m

```diff
@@ -278,6 +283,7 @@
 NSString *const BOXAPIObjectKeyURL = @"url";
 NSString *const BOXAPIObjectKeyURLTemplate = @"url_template";
 NSString *const BOXAPIObjectKeyDownloadURL = @"download_url";
+NSString *const BOXAPIObjectKeyAuthenticatedDownloadURL = @"authenticated_download_url";
```

Comment:
Can't documentation on this key. Its not used in https://github.com/box/box-ios-preview-sdk-dev/pull/890/files yet, but it is something that is a tbd?

Reply:
Correct, we discovered from the reps team that original files are served either from 'authenticated_download_url' or 'download_url'. They are both ok to use, for due diligence, I added both. The difference is authenticated_download_url supported caching for web.